packages/EasyCore/src/Bridge/Symfony/ApiPlatform/Normalizers/CarbonImmutableNormalizer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Bridge\Symfony\ApiPlatform\Normalizers;
+
+use Carbon\CarbonImmutable;
+use Symfony\Component\Serializer\Normalizer\DateTimeNormalizer;
+
+final class CarbonImmutableNormalizer extends DateTimeNormalizer
```

Comment:
Please extend the following method `\Symfony\Component\Serializer\Normalizer\DateTimeNormalizer::supportsDenormalization`